src/vendor.browser.ts

```diff
@@ -12,9 +14,36 @@ import '@angular/forms';
 import '@angular/http';
 import '@angular/router';
 
+// Angular2 Material
+import '@angular2-material/core';
```

Comment:
Why this? I did not include this in my update and it works: https://github.com/Sirimangalo/meditation-plus-angular/commit/824c60bd2dbb4063e869920f13e93539af3d453d